README.md

```diff
@@ -0,0 +1,21 @@
+## Setup instructions
+
+```sh
+git clone https://github.com/InwithSid/B5-Asessment-2nd.git B5-Assessment-2nd
```

Comment:
This, perhaps need to be dynamic, because this repo is meant to be used as a template. I am not sure how to achieve that. 🤔